app/views/questions/index.html.erb

```diff
@@ -1,28 +1,65 @@
 <h1 class="questions_header">All Questions</h1>
 
-  <table class="striped">
-    <thead>
-      <tr>
-        <th>Votes</th>
-        <th>Answers</th>
-        <th>Title</th>
-        <th>Creator</th>
-        <th>Category</th>
-        <th>Updated_at</th>
-      </tr>
-     </thead>
+<div class="row">
```

Comment:
Thanks for taking the time to keep all of this formatted nicely; it makes it significantly easier to read through. Looks awesome also!